receiver/awscontainerinsightreceiver/receiver.go

```diff
@@ -169,6 +176,30 @@ func (acir *awsContainerInsightReceiver) startPrometheusScraper(ctx context.Cont
 	})
 	return err
 }
+func (acir *awsContainerInsightReceiver) startDcgmScraper(ctx context.Context, host component.Host, hostinfo *hostInfo.Info) error {
```

Comment:
nit: I know the prometheus one is named this way, but is this really starting the scrapper? Aren't we relying on `GetMetrics` to start it?

Reply:
good call. Updated to `init` instead of `start`